source/offboarding.rst

```diff
@@ -0,0 +1,144 @@
+.. _section-offboarding:
+
+Offboarding
+===========
+
+This page contains instructions on how to remove the Digital Humanities Innovation Lab's 
+developing environment from your computer. The current instructions are only for Mac computers.
+Tools are presented in the order they appear in the developing environment set-up guide.
+
+Xcode Command Line Tools
+------------------------
+
+If you installed the XCode command line tools following the instructions in this guide, you can
+uninstall it by removing the ``/Library/Developer/CommandLineTools`` directory. The command looks like
+
+.. code-block:: console
+
+  $ sudo rm -rf /Library/Developer/CommandLineTools
+
+You will need to run this command using sudo, which will prompt you for your administrator password.
+
+Homebrew
+--------
+
+You can uninstall Homebrew using the uninstall command that is printed on the `Homebrew FAQs`_ page. 
+It looks like
+
+.. code-block:: console
+
+  $ ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/uninstall)"
+
+That command will attempt to uninstall and remove all homebrew pacakages. If it does not completely 
+remove all packages, you may need to run the command using sudo, which will prompt you for your 
+administrator password.
+
+.. _Homebrew FAQs: https://docs.brew.sh/FAQ
+
+Sphinx and Python
+-----------------
+
+As python and Sphinx are installed using Homebrew, if you remove all Homebrew packages and uninstall 
+Homebrew, python and Sphinx will also be uninstalled. However, if you wish to keep Homebrew and only 
+remove python and Sphinx, you can do so using the following command
+
+.. code-block:: console
+
+  $ brew uninstall python --force
+
+``--force`` is required to ensure all Homebrew versions of python are uninstalled. Otherwise, 
+Homebrew will continue to install the latest version of python (or any package) that it knows 
+about when you update all packages using ``brew upgrade``.
+
+As Sphinx is a python package, if you wish you keep Homebrew and the Homebrew installation of 
+Python but remove Sphinx, you can do so using python's package manager. The command to do so is
+
+.. code-block:: console
+
+  $ pip3 uninstall Sphinx
```

Comment:
If you uninstall Python, will it uninstall pip3? If that's the case, then you'd need to do this before uninstalling python.